sim/druid/item_sets_pve_phase_4.go

```diff
@@ -225,7 +226,8 @@ var ItemSetCenarionRage = core.NewItemSet(core.ItemSet{
 		},
 		// Reduces the cooldown of Enrage by 30 sec and it no longer reduces your armor.
 		4: func(agent core.Agent) {
-			// TODO: Enrage
+			druid := agent.(DruidAgent).GetDruid()
+			druid.Enrage.CD.Duration -= time.Second * 30
```

Comment:
This changed since you put up the PR but we should modify the new CD `FlatModifier` field for effects like this now. CDs store the original Duration, a cumulative flat modifier like this one, and then a cumulative multiplier, then the sim calculates the real duration from those like the game does

```suggestion
			druid.Enrage.CD.FlatModifier -= time.Second * 30
```

Reply:
Oh also since this is also a rune, we'll want to follow the pattern of creating a function that does the aura check / creates an aura to handle the set bonus. Following the other example, create the aura, then in the `OnInit` you can make the change to Enrage's CD FlatModifier